src/main/scala/io/xskipper/metadatastore/MetadataStoreManager.scala

```diff
@@ -28,6 +28,9 @@ trait MetadataStoreManager {
     */
   def getType: MetadataStoreManagerType
 
+
+  protected var objectStatsEnabled: Boolean = true
```

Comment:
please add documentation on why this is needed both to the site and the PR and reference the relevant issue